homeassistant/components/cloud/google_config.py

```diff
@@ -122,7 +127,7 @@ def __init__(
         config: dict[str, Any],
         cloud_user: str,
         prefs: CloudPreferences,
-        cloud: Cloud,
+        cloud: Cloud["CloudClient"],
```

Comment:
Add `from __future__ import annotations` to the top of the file, and you can change this to be like the others.
```suggestion
        cloud: Cloud[CloudClient],
```